app/models/split_upsert.rb

```diff
@@ -1,13 +1,13 @@
 class SplitUpsert
   include ActiveModel::Model
 
-  attr_accessor :app, :name, :decided_at, :finished_at, :require_app_name_prefix
+  attr_accessor :app, :name, :owner, :decided_at, :finished_at, :require_app_name_prefix
   attr_reader :weighting_registry
 
   validate :split_must_be_valid
 
   def save # rubocop:disable Metrics/AbcSize
-    split.assign_attributes(finished_at: finished_at, decided_at: decided_at, require_app_name_prefix: require_app_name_prefix)
+    split.assign_attributes(finished_at: finished_at, decided_at: decided_at, require_app_name_prefix: require_app_name_prefix, owner: owner)
```

Comment:
In the case where the client is not yet sending an `owner` param, do we want to do something to avoid overwriting an existing owner with `nil`?

Reply:
(given that this is an "upsert" operation)